### PR TITLE
Features/pktz4-favorite-view-UI

### PR DESCRIPTION
**Pull request by mazeneldeeb9**
finish favorite view

### CODE REVIEW

File: Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+struct FavoriteCard: View {
+    let movie: Movie
+    @State private var isFavorite = true
+    var body: some View {
+        ZStack {
+            Rectangle().frame(
+                width: .infinity, height: 170)
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift]: dont put width: infinity

File: Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    isFavorite.toggle()
+                }, label: {
+                    Image(systemName: isFavorite ? "heart.fill" : "heart")
+                        .foregroundStyle(.red)
+                        .font(/*@START_MENU_TOKEN@*/.title/*@END_MENU_TOKEN@*/)
+                }).padding(.trailing, 25)
+                    .padding(.bottom, 60)
+            }
+        }
+        .frame(width: .infinity, height: 200)
```
Comment (MohammedAbdullaiOS) [inline Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift]: dont put width infinity

Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift]: done

File: Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift (hunk 1 above)
Comment (mazeneldeeb9) [reply Peek-swiftUI/Modules/FavoriteView/views/FavoriteCard.swift]: done

**Pull request merged**